src/components/NavBar/NavBar.vue

```diff
@@ -88,7 +143,26 @@ export default {
       isLoading: false,
       searchModel: null,
       search: null,
-      searchQueue: []
+      searchQueue: [],
+      notifItems: [
+        { avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg', message: 'Ali Connors I\'ll be in your neighborhood doing errands this weekend. Do you want to hang out?', date: '3/12/2021, 7:14:33 PM', readed: false },
+        { message: 'to Alex, Scott, Jennifer: Wish I could come, but I\'m out of town this weekend.', date: '1/11/2019, 7:14:33 PM', readed: false },
```

Comment:
```suggestion
        { message: 'to Alex, Scott, Jennifer: Wish I could come, but I\'m out of town this weekend.', date: '1/11/2019, 7:14:33 PM', read: false },
```
I'd have read/unread instead ("readed" isn't a word ;) )

Reply:
its fixed!